content/v8.0.0/index.yml

```diff
@@ -0,0 +1,34 @@
+layout: event
+
+header: /_shared/header.md
+
+type: event
+pagetitle: Design System meetup Sydney v8.0.0
+version: v8.0.0
+city: Sydney
+date: 2019-05-13
+location: /_locations/hipages.md
+sponsors:
+  - /_sponsors/canva.md
+  - /_sponsors/thinkmill.md
+  - /_sponsors/tomwalker.md
+link: https://www.meetup.com/Design-System/events/258516904
```

Comment:
Just caught - this is the url for 7.0.0 meetup - needs to be updated too.

Reply:
fixed